fix(18448): recolor selected area point geometry

alesiahil opened the pull request:
Tasks:
https://kontur.fibery.io/Tasks/Task/Recolor-point-geometry-into-blue-for-Selected-area-18448
https://kontur.fibery.io/Tasks/Task/Fix-labels-and-texts-on-Kontur-apps-18566

alesiahil committed:
Update FocusedGeometryRenderer.ts

albaranau commented on src/features/focused_geometry_layer/renderers/FocusedGeometryRenderer.ts:
```diff
@@ -1,5 +1,5 @@
 import { UserGeometryRenderer } from '~core/logical_layers/renderers/UserGeometryRenderer';
-import Icon from '../icons/marker_black.png';
```
The `marker_black.png` file itself doesn't seem to be used anywhere after this change. Would be better to delete it

alesiahil replied:
I assume it is used in edit mode

albaranau replied:
The black marker in edit mode comes from a different file (iconAtlas.png)

alesiahil replied:
Done

alesiahil committed:
Update common.json

Fix the missed part of https://kontur.fibery.io/Tasks/Task/Sort-HOT-Projects-panel-by-project-number-by-default-18224

alesiahil committed:
Update common.json

Removing the article 'the' for the button 'Set reference area' in user profile

alesiahil committed:
Delete src/features/focused_geometry_layer/icons/marker_black.png

alesiahil committed:
Update some labels and wordings

Komzpa commented on src/core/localization/translations/en/common.json:
```diff
@@ -2,13 +2,13 @@
   "disasters": "Disasters",
   "loading": "Loading...",
-  "loading_events": "Loading events",
+  "loading_events": "Loading disasters",
```
@palina-krukovich review this change please.

palina-krukovich replied:
looks good

alesiahil committed:
Update common.json

Fixed some wordings

alesiahil committed:
Update common.json

albaranau commented on src/core/localization/translations/en/common.json:
```diff
@@ -287,10 +287,10 @@
-      "email_invalid": "Email has to be valid!",
-      "password": "Password has not to be empty!",
-      "connect": "Couldn't connect to authentication service"
+      "email_empty": "Email cannot to be empty",
```
"Email cannot be empty"

albaranau commented on src/core/localization/translations/en/common.json:
```diff
@@ -287,10 +287,10 @@
-      "connect": "Couldn't connect to authentication service"
+      "email_empty": "Email cannot to be empty",
+      "email_invalid": "Email has to be valid",
+      "password": "Password cannot to be empty",
```
"Password cannot be empty"

alesiahil committed:
Update common.json

alesiahil committed:
Update common.json

albaranau commented on src/core/localization/translations/en/common.json:
```diff
@@ -358,11 +358,11 @@
   },
   "current_event": {
-    "not_found_request": "Sorry, requested event not found."
+    "not_found_request": "Sorry, requested disaster is not found."
```
Are we sure "is not found" works here? Present tense feels weird for some reason. Maybe "was not found" would work better, or just "not found"

Komzpa replied:
Here are a few improved options for the message:

"Sorry, the requested disaster could not be found."
"Apologies, the requested disaster information is unavailable."
"Sorry, we couldn't locate the requested disaster."
"We apologize, but the disaster you requested is not found."
"The requested disaster information is not available. Please try again."

alesiahil committed:
Update common.json

alesiahil committed:
Update common.json

alesiahil commented on src/core/localization/translations/en/common.json:
```diff
@@ -173,16 +173,16 @@
-    "not_found": "Sorry, requested event not found.",
-    "error": "An error occured!",
-    "analytics_for_selected": "Analytics for a selected area",
+    "not_found": "Sorry, the requested disaster was not found.",
```
@hevanskontur or @gmdowlingkontur can you take a look at this file https://github.com/konturio/disaster-ninja-fe/blob/9055f60ddff3a9bbc201f02584f250e16d57ede3/src/core/localization/translations/en/common.json (except 'about Disaster Ninja' part) and add your comments if there are any mistakes?

alesiahil committed:
Update toolBarTexts.spec.ts

alesiahil committed:
Update toolBarTextsWithUser.spec.ts

alesiahil committed:
Update profilePage.ts

alesiahil committed:
Update login.spec.ts

alesiahil committed:
Update common.json

alesiahil committed:
Update common.json

alesiahil committed:
Update common.json

alesiahil committed:
Update common.json
